test/arrayweeks_test.jl

```diff
@@ -0,0 +1,70 @@
+
+function arrFcomplex(s)
+    # Laplace domain
+    α = complex(-0.3, 6.0)
+    return [1 2;3 4] ./ (s - α)
+end
+
+function Fcomplex(s)
+    # Laplace domain
+    α = complex(-0.3, 6.0)
+    return 1 / (s - α)
+end
+
+# Test scalar-functionality of arrcoeff, Fcomplex=(s-a)⁻¹
+let arr = try InverseLaplace._arrcoeff(Fcomplex,4,1.0,1.0);
+    InverseLaplace._arrcoeff(Fcomplex,4,1.0,1.0)
+    catch
+        InverseLaplace._arrcoeff(Fcomplex,4,1.0,1.0)
+    end, scal = InverseLaplace._wcoeff(Fcomplex,4,1.0,1.0)
+
+        @test arr == scal
+end
+
+# Test for ordering of array valued coefficients (along first dimension)
+let arr = try InverseLaplace._arrcoeff(arrFcomplex,4,1.0,1.0);
+    InverseLaplace._arrcoeff(arrFcomplex,4,1.0,1.0)
+    catch
+        InverseLaplace._arrcoeff(arrFcomplex,4,1.0,1.0)
```

Comment:
Is it necessary to test `_arrcoeff` if `_get_array_coefficients` is tested as well?

Reply:
No, it shouldn't be necessary. I don't know whats up. 100 % coverage before is suspicious. Still, it may be accurate.